src/libcore/iter/mod.rs

```diff
@@ -1031,17 +1031,15 @@ impl<I: ExactSizeIterator> ExactSizeIterator for Peekable<I> {}
 impl<I: Iterator> Peekable<I> {
     /// Returns a reference to the next() value without advancing the iterator.
     ///
-    /// The `peek()` method will return the value that a call to [`next()`] would
-    /// return, but does not advance the iterator. Like [`next()`], if there is
```

Comment:
Output isn't just for HTML, eventually we will have things like "look up this method on the command line", which many languages have and many people ask for.